model/task_queue.go

```diff
@@ -555,17 +555,6 @@ func FindAllTaskQueues() ([]TaskQueue, error) {
 func FindDistroTaskQueue(distroID string) (TaskQueue, error) {
 	queue := TaskQueue{}
 	err := db.FindOneQ(TaskQueuesCollection, db.Query(bson.M{taskQueueDistroKey: distroID}), &queue)
-
-	grip.DebugWhen(err == nil, message.Fields{
```

Comment:
All the debug logs in this file + the scheduler look like leftovers from the tunable scheduler project. I can't find any actively-used dashboard that uses this info.